.buildbot.sh

```diff
@@ -6,18 +6,18 @@
 
 set -e
 
-export CHERIBASE=~/cheri
+CHERI_DIR=/home/buildbot/cheri/output
 CHERIBUILD=~/build
-export TEST_TARGET=$1
 
 echo "Checking clang-format..."
 # Note that `sdk` and `morello-sdk` contain different versions of clang-format,
 # so we have to pick one and use it consistently.
-CLANG_FORMAT="$CHERIBASE"/output/sdk/bin/clang-format
+CLANG_FORMAT=$CHERI_DIR/morello-sdk/bin/clang-format
 find . -iname "*.c" -o -iname "*.h" -o -iname "*.cpp" -o -iname "*.hpp" | xargs "$CLANG_FORMAT" --dry-run -Werror
 
 echo "Checking that all the purecap examples build for all platforms..."
-PLATFORMS='riscv64-purecap morello-purecap'
+#PLATFORMS='riscv64-purecap morello-purecap'
+PLATFORMS='morello-purecap'
```

Comment:
Shouldn't we be testing more than just morello-purecap?

Reply:
Right, forgot to add this in. Our nightly doesn't yet build risc-v, so it'll take a while to rebuild and test.

Reply:
No problem!

Reply:
While I'm here, do we also want to build RISC-V hybrid, or just purecap?

Reply:
I think we previously only tested riscv-purecap? If so, that's good enough for now (we can add risc-v hybrid a bit later).

Reply:
Re-added in https://github.com/capablevms/cheri-examples/pull/68/commits/8a34e3adaab6b1001017cf843f9a3d80e204caa7.